packages/web-app-draw-io/src/index.ts

```diff
@@ -28,38 +37,53 @@ export default defineWebApplication({
       }
     ]
 
-    return {
-      appInfo: {
-        name: 'Draw.io',
-        id: applicationId,
-        icon: 'grid',
-        color: '#EF6C00',
-        applicationMenu: {
-          enabled: () => {
-            return !!userStore.user
-          },
-          priority: 30,
-          openAsEditor: true
-        },
-        defaultExtension: 'drawio',
-        extensions: [
-          {
-            extension: 'drawio',
-            routeName: 'draw-io',
-            newFileMenu: {
-              menuTitle() {
-                return $gettext('Draw.io document')
-              }
+    const appInfo: ApplicationInformation = {
+      name: 'Draw.io',
+      id: applicationId,
+      icon: 'grid',
+      color: '#EF6C00',
+      defaultExtension: 'drawio',
+      extensions: [
+        {
+          extension: 'drawio',
+          routeName: 'draw-io',
+          newFileMenu: {
+            menuTitle() {
+              return $gettext('Draw.io document')
             }
-          },
-          {
-            extension: 'vsdx',
-            routeName: 'draw-io'
           }
-        ]
-      },
+        },
+        {
+          extension: 'vsdx',
+          routeName: 'draw-io'
+        }
+      ]
+    }
+
+    const menuItems = computed<AppMenuItemExtension[]>(() => {
+      const items: AppMenuItemExtension[] = []
+
+      if (userStore.user) {
+        items.push({
+          id: `app.${appInfo.id}.menuItem`,
+          type: 'appMenuItem',
+          label: () => appInfo.name,
+          color: appInfo.color,
+          icon: appInfo.icon,
+          priority: 30,
```

Comment:
do we have some guideline how we define priorities?

Reply:
I guess the best description is currently in the [external-sites README](https://github.com/owncloud/web-extensions/tree/main/packages/web-app-external-sites), but it should be part of https://owncloud.dev/clients/web/extension-system/extension-types/app-menu-items/#configuration.